### PR TITLE
Fix: Blizzard Lighting Pixellicious  35 Channel Mode

### PR DESCRIPTION
**Pull request by yestalgia**

### CODE REVIEW

**Comment by mcallegari** [comment]
Hi, thanks. What about the 480 channel mode?

**Comment by yestalgia** [comment]
> Hi, thanks. What about the 480 channel mode?
No problem, I will add the additional channels. 160 heads here we go!

File: resources/fixtures/Blizzard_Lighting/Blizzard-Lighting-Pixellicious.qxf [hunk 1]
```diff
@@ -147,6 +633,1288 @@
   <Channel Number="6">Auto Programs 16-29 + Mix Mode</Channel>
   <Channel Number="7">Auto Program Speed</Channel>
  </Mode>
+ <Mode Name=" 480 Channel Mode">
```
Comment (mcallegari) [inline resources/fixtures/Blizzard_Lighting/Blizzard-Lighting-Pixellicious.qxf:636]: Please remove the first space and the word "Mode"

**Comment by mcallegari** [comment]
Thanks :+1:

**Pull request merged**